local-storage.js

```diff
@@ -0,0 +1,15 @@
+const formElements = document.querySelectorAll('.storage');
+const stringValues = localStorage.getItem('data');
+const parseValues = JSON.parse(stringValues);
+
+[formElements[0].value, formElements[1].value, formElements[2].value] = parseValues;
+for (let i = 0; i < formElements.length; i += 1) {
+  formElements[i].addEventListener('keyup', () => {
+    const nameValue = formElements[0].value;
+    const emailValue = formElements[1].value;
+    const textAreaValue = formElements[2].value;
+    const values = [nameValue, emailValue, textAreaValue];
+    const stringifyValues = JSON.stringify(values);
+    localStorage.setItem('data', stringifyValues);
+  });
+}
```

Comment:
Please, ensure that your app is working perfectly as it has been prescribed in the requirement 🙃.

- Your Functionality
![mobile wrong](https://user-images.githubusercontent.com/41163571/224503720-dcd2cfdb-6981-4525-a0af-2013073508b0.gif)

- Required Functionality
![mobile right](https://user-images.githubusercontent.com/41163571/224503748-3bc4bb74-05a4-491c-bdea-47f0154bc3d6.gif)
😀 
The data in the inputs are meant to be persisted even after refresh.
Kindly use [this](https://www.youtube.com/watch?v=AUOzvFzdIk4), [this](https://www.youtube.com/watch?v=k8yJCeuP6I8) and [this](https://www.youtube.com/watch?v=fYTTUBa-lPc) videos for better understanding of the project topic.

you can likewise use `window.onload` to input the data from the local storage into the `input` 😃.

Reply:
Hello @AdnanOlarmmi Im surprised because everything works fine on my machine. I have also tried using different browsers and all works fine